docker/build.sh

```diff
@@ -184,11 +184,6 @@ function docker_cmd()
             DOCKER_TAG_TESTS=${GIT_SHA:0:12}
         fi
 
-        # builds the juptyer notebooks and rstudio docker images that go on dataproc clusters
-        # Stop building jupyter and rstudio images in leonardo repo as we move the process to terra-docker repo
-#        bash ./docker/build-tool.sh -i jupyter -r "${NOTEBOOK_REPO}" -t "${DOCKER_TAG}"
-#        bash ./docker/build-tool.sh -i rstudio -r "${NOTEBOOK_REPO}" -t "${DOCKER_TAG}"
-
         # Build the UI if specified.
         if $BUILD_UI; then
           bash ./ui/build.sh build "${NOTEBOOK_REPO}" "${DOCKER_TAG}"
```

Comment:
would be nice to update this script to only publish leonardo images to gce only as well at some point

Reply:
Yeah.. I feel like `build.sh` could be simplified a lot, maybe best done as a separate PR